src/cmds/versions/create.ts

```diff
@@ -10,14 +10,16 @@ import fetch, { cleanHeaders, handleRes } from '../../lib/fetch';
 import * as promptHandler from '../../lib/prompts';
 import promptTerminal from '../../lib/promptWrapper';
 
-export type VersionCreateOptions = { fork?: string } & VersionBaseOptions;
+export interface Options extends CommonOptions {
```

Comment:
Renamed this and `VersionBaseOptions` so they're consistent with our other commands.